src/v2/wallet.ts

```diff
@@ -1047,25 +1047,29 @@ Wallet.prototype.removeUser = function(params, callback) {
  * @param {Boolean} params.instant - Build this transaction to conform with instant sending coin-specific method (if available)
  * @param {{value: String, type: String}} params.memo - Memo to use in transaction (supported by Stellar)
  * @param {String} params.addressType - The type of address to create for change. One of `p2sh`, `p2shP2wsh`, and `p2wsh`. Case-sensitive.
+ * @param {Boolean} params.hop - Build this as an Ethereum hop transaction
+ * @param {String} params.walletPassphrase The passphrase to the wallet user key, to sign commitment data for Ethereum hop transactions
  * @param callback
  * @returns {*}
  */
 Wallet.prototype.prebuildTransaction = function(params, callback) {
   return co(function *() {
-    // Whitelist params to build tx (mostly around unspent selection)
+    // Whitelist params to build tx
     const whitelistedParams = _.pick(params, [
       'recipients', 'numBlocks', 'feeRate', 'maxFeeRate', 'minConfirms', 'enforceMinConfirmsForChange',
       'targetWalletUnspents', 'message', 'minValue', 'maxValue', 'sequenceId', 'lastLedgerSequence',
       'ledgerSequenceDelta', 'gasPrice', 'noSplitChange', 'unspents', 'changeAddress', 'instant', 'memo', 'addressType',
-      'cpfpTxIds', 'cpfpFeeRate', 'maxFee', 'idfVersion', 'idfSignedTimestamp', 'idfUserId', 'strategy'
+      'cpfpTxIds', 'cpfpFeeRate', 'maxFee', 'idfVersion', 'idfSignedTimestamp', 'idfUserId', 'strategy', 'hop'
     ]);
     debug('prebuilding transaction: %O', whitelistedParams);
 
     if (params.reqId) {
       this.bitgo._reqId = params.reqId;
     }
+    const extraParams = yield this.baseCoin.getExtraPrebuildParams(Object.assign(params, { wallet: this }));
+    const buildParams = Object.assign(whitelistedParams, extraParams);
```

Comment:
Nit: Don't need this extra var. Since Object.assign mutates it's target, you can continue using `whitelistedParams` instead of aliasing it as `buildParams`.

Reply:
oof didnt know it mutated